great-idea-website/index.html

```diff
@@ -13,51 +13,73 @@
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
   
+  <body>
+  <main>
+    <header>
+      <nav>
+        <a href='#'>Services</a>
+        <a href='#'>Product</a>
+        <a href='#'>Vision</a>
+        <a href='#'>Features</a>
+        <a href='#'>About</a>
+        <a href='#'>Contact</a>
+      </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
+
+    <section class="tagline">
+      <h1>Innovation</h1>
+      <h1>On</h1>
+      <h1>Demand</h1>
+      <a class="calltoaction" href='#'>Get Started</a>
+    </section>
+
+      <img class="headerimg" src="img/header-img.png" alt="Image of a code snippet.">
+      
+    <hr>
+    <section class="twocolumn">
+      <h3>Features</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+    </section> 
+    <section class="twocolumn">
+      <h3>About</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>   
+    </section> 
+    
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    
+    <section class="threecolumn"> 
+      <h3>Services</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+    </section>
+    <section class="threecolumn">
+      <h3>Product</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+    </section>
+    <section class="threecolumn">
+      <h3>Vision</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+    </section>
+    <hr>
+    <section>
+      <h3>Contact</h3>
+      <address>
+      <p>c123 Way 456 Street
+      Somewhere, USA</p>
+      </address>
+      <p>1 (888) 888-8888</p>
+      <p>sales@greatidea.io</p>
+    </section>
+
+    <footer>
+      <p>Copyright Great Idea! 2018</p>
+    </footer>
+  </main>
   </body>
 </html>
```

Comment:
Overall, looking like a great project.  The design is pretty much matching up to the design spec.  Change the a tag to a button tag for the get started button, but a really good job!